### PR TITLE
fix and bump rubocop

### PR DESCRIPTION
**Pull request by hjanuschka**
fixes new issues with rubocop => 0.47 👮

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -38,6 +38,10 @@ Style/NumericLiteralPrefix:
 Style/TernaryParentheses:
   Enabled: false
 
+# sometimes it is usefull to have those empty methods
```
Comment (lacostej) [inline .rubocop.yml]: Here I think rubocop just wants us to do something like:

```
      def self.return_value; end
```

I am not against it. It's a few files to change.

File: fastlane/lib/fastlane/actions/appaloosa.rb [hunk 2]
```diff
@@ -83,7 +83,11 @@ def self.get_screenshots_links(api_key, store_id, screenshots_path, locale, devi
       def self.get_screenshots(screenshots_path, locale, device)
         get_env_value('screenshots').nil? ? locale = '' : locale.concat('/')
         device.nil? ? device = '' : device.concat('-')
-        !screenshots_path.strip.empty? ? screenshots_list(screenshots_path, locale, device) : nil
+        if !screenshots_path.strip.empty?
```
Comment (lacostej) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: I think it meant it preferred to invert the condition into something like
````
screenshots_path.strip.empty? ? nil : screenshots_list(screenshots_path, locale, device)
````

**Review by lacostej — COMMENTED** [review]
@hjanuschka see my comments! Thanks for the PR

File: fastlane/lib/fastlane/actions/ipa.rb [hunk 3]
```diff
@@ -1,5 +1,4 @@
 # rubocop:disable Lint/AssignmentInCondition
-# rubocop:disable Style/Next
```
Comment (mfurtak) [inline fastlane/lib/fastlane/actions/ipa.rb]: Remove matching tag from end of file

File: .rubocop.yml (hunk 1 above)
Comment (hjanuschka) [inline .rubocop.yml]: yeah `-a` would fix it anyway 👍  i am not a fan of it, and i thought that i saw some other rules  like that.

Comment (mfurtak) [inline .rubocop.yml]: I am in favor of having this check disabled. :+1: I don't find the one-line style to be major win.

File: fastlane/lib/fastlane/actions/ipa.rb (hunk 3 above)
Comment (hjanuschka) [inline fastlane/lib/fastlane/actions/ipa.rb]: done - thx

File: fastlane/lib/fastlane/actions/appaloosa.rb (hunk 2 above)
Comment (hjanuschka) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: 👍  thx - fixed

**Review by mfurtak — APPROVED** [review]

**Pull request merged**